### DISCUSSION TITLE
A question about custom controls for WinUI 3

### DISCUSSION THREAD

**Discussion by AFunDog** [CommunityToolkit/Windows#449] [Q&A]
I am a beginner.
I want to know how to package custom controls into the Nuget package so that I can use them in other projects.

**Comment by ShimizuTheLotus** [comment]
I hope this tutorial will help you:
https://learn.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio?tabs=netcore-cli
